docs/en/advanced-concepts/migrating-to-xcuitest.md

```diff
@@ -0,0 +1,44 @@
+## Migrating your iOS tests from UIAutomation (iOS 9 and below) to XCUITest (iOS 9 and up)
```

Comment:
I know that technically it is iOS 9 and above, but functionally, for Appium users, it is 9.3 and above.